resources/charts/stolon/templates/proxy-deployment.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "stolon.fullname" . }}-proxy
+  labels:
+    app: {{ template "stolon.name" . }}
+    chart: {{ template "stolon.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.proxy.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "stolon.name" . }}
+      release: {{ .Release.Name }}
+      component: stolon-proxy
+  template:
+    metadata:
+      labels:
+        app: {{ template "stolon.name" . }}
+        release: {{ .Release.Name }}
+        component: stolon-proxy
+        stolon-cluster: {{ template "stolon.fullname" . }}
+      annotations:
+{{- with .Values.proxy.annotations }}
+{{ toYaml . | indent 8 }}
+{{- end }}
+    spec:
+{{- if .Values.proxy.priorityClassName }}
+      priorityClassName: "{{ .Values.proxy.priorityClassName }}"
+{{- end }}
+      serviceAccountName: {{ template "stolon.serviceAccountName" . }}
+      initContainers:
+        - name: fix-secrets
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/usr/bin/dumb-init", "--", "/bin/bash", "-exc"]
+          args:
+            - |
+              if [ -d /etc/secrets/etcd ]; \
+              then \
+                  cp -R /etc/secrets/etcd /home/stolon/secrets/etcd; \
+                  chown -R stolon /home/stolon/secrets/etcd; \
+              fi
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: stolon-secrets
+              mountPath: /home/stolon/secrets
+            {{- with .Values.proxy.initVolumeMounts }}
+{{ toYaml . | indent 12 }}
```

Comment:
The absolute indent basically jumps at you. Please take a look at [this](https://github.com/helm/helm/pull/3837) and see if this can be improved.